src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -222,6 +222,33 @@ def _check_and_set_dashboard(self,
             except MonCommandFailed as e:
                 logger.warning('Failed to set Dashboard config for %s: %s', service_name, e)
 
+    def ok_to_stop_osd(
+            self,
+            osds: List[str],
+            known: Optional[List[str]] = None,
+            force: bool = False) -> HandleCommandResult:
+        r = HandleCommandResult(*self.mgr.mon_command({
+            'prefix': "osd ok-to-stop",
+            'ids': osds,
+            'max': 16,
+        }))
+        j = None
+        try:
+            j = json.loads(r.stdout)
+        except json.decoder.JSONDecodeError:
+            self.mgr.log.warning("osd ok-to-stop didn't return structur result")
+            pass
```

Comment:
```suggestino
raise
```

I'd be :+1: for seeing tracker issues, if `ok-to-stop`  returns something invalie.